### ISSUE TITLE
Consolidate some models into a single model using submodels

### ISSUE THREAD

**Issue by SaintWish** [MSRevive/assets#40]
It would be really helpful to consolidate similar models into a single file by just making them submodels. This would decrease the amount of models we would have to precache.

**Comment by tschumann** [comment]
Are the model sources available for most models?

**Comment by SaintWish** [comment]
No, we're missing a majority of model sources if not all of them.

Most of the models are already consolidated together, but I think there's some that could still.
